samples/spring-customer-registry-views-quickstart/pom.xml

```diff
@@ -57,38 +53,6 @@
         </configuration>
       </plugin>
 
-      <plugin>
-        <groupId>org.xolstice.maven.plugins</groupId>
-        <artifactId>protobuf-maven-plugin</artifactId>
```

Comment:
I initially thought we needed it to generate some protobuf used inside. But that's only the SDK that needs it. The final user project doesn't.